tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -88,6 +88,13 @@ define(KWD_PIPE_SCH_DEADLINE_US, 20000)
 # include the generic dmic with kwd
 include(`platform/intel/intel-generic-dmic-kwd.m4')
 
+# BT offload support
+define(`BT_PIPELINE_PB_ID', eval(SMART_REF_PPL_ID + 1))
+define(`BT_PIPELINE_CP_ID', eval(SMART_REF_PPL_ID + 2))
+define(`BT_PIPELINE_BE_ID', eval(SMART_BE_ID + 1))
```

Comment:
Is this defining the PCM device number? If yes, can we try and standardize what the value is, instead of having is variable based on whatever else is on this platform? The name of the PCM devices also matter

For SoundWire this is what we used
````
PCM13  playback Bluetooth
PCM14  capture 	Bluetooth
````

Reply:
The user has to define the following variables:

- pipe id for playback/capture
- link id

We either add at the end or insert in the middle depends on the m/c link creation order.
The pcm id is hard-coded for 50, just a distinctive value. 

Do you prefer to standardize all above with reserved values? thanks!!

Reply:
@plbossart I wonder if it's better to still allow the caller to define the PCM IDs, and not hardcode it intel-generic-bt.m4. We might have standard number in all SDW systems, but if there is need to deviate, it's seems messy if we have hardcoded values in the "pipeline helper".

Reply:
@plbossart btw, have we published the SDW PCM mappings anywhere yet ?

Reply:
> @plbossart btw, have we published the SDW PCM mappings anywhere yet ?

no, I'll have to figure out where to publish.

Reply:
> At least agree on names so that UCM can look for the PCM device. IIRC Jaroslav added a find-by-name option.

Should we change the name from "BT-PCM" to "Bluetooth"? suggestions are welcome :)

``card 0: sofrt5682 [sof-rt5682], device 50: BT-PCM (*) []``
``Subdevices: 1/1``
`` Subdevice #0: subdevice #0``

Reply:
I'm ok with simply "Bluetooth" as well. "BT-PCM" is probably good enough to be used as a UCM key, but for casual user looking at alsa-info output, or debugging UCM, "Bluetooth" can be easiar to understand. @plbossart which gets your preference?

Reply:
We discussed this a while back, for now the idea was to use a generic 'Bluetooth' PCM name, if and when we have different devices for different profiles we would add more specific names.

but don't give it the '50' number. that makes no sense.

Reply:
Ack, will adopt generic 'Bluetooth' PCM name in v5, for the pcm id, can you recommend a fixed value as default? or a sequential number is preferred? Thanks!

Reply:
something 'reasonably sequential', e.g. last HDMI device + 2 would probably work, but like I said above the number is less important if you can search by name.